src/main.rs

```diff
@@ -15,15 +15,20 @@
 // Written by: Bobbbay Bobbayan
 
 use std::{thread, time, fs, env};
-
+use std::time::{SystemTime, UNIX_EPOCH};
 fn main() {
     let args: Vec<String> = env::args().collect();
 
+    let start = SystemTime::now();
+    let since_the_epoch = start
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards");
```

Comment:
Give a more generic error - "The time went backwards" might be misleading in the 10% scenario where it's not about that error.